docs/intro.md

```diff
@@ -1,8 +1,7 @@
-### Operate First - Getting Started
+### Getting started with Operate First
 
-#### This readme contains a getting started guide to Operate First.
-
-If you are interested in bringing your project to the Operate First Environment at the Mass Open Cloud (MOC) you have come to the right place. Below is important information and instructions on how to get started.
+If you are an open source developer interested in bringing your project to an Operate First environment at the Mass Open Cloud (MOC), you have come to the right place.
```

Comment:
`at the Mass Open Cloud (MOC)` should we remove this part? since we have other environments as well now.

Reply:
@HumairAK @tumido